Tools/CISupport/build-webkit-org/steps_unittest.py

```diff
@@ -413,12 +413,12 @@ def test_success(self):
 
     def test_bigsur_timeout(self):
```

Comment:
I think we can remove this test entirely given your change to `steps.py`

Reply:
That caused webkitpy tests to fail, but only on EWS.  I couldn't hit the problem locally (on Sonoma, admittedly)